lib/tasks/deployment/20190918115440_set_up_long_furlong.rake

```diff
@@ -0,0 +1,16 @@
+namespace :after_party do
+  desc 'Deployment task: set_up_long_furlong'
+  task set_up_long_furlong: :environment do
```

Comment:
Not sure about putting live school data in here. Also means we can't do the setup in the admin interface on test?

Reply:
We can leave it out for now, the UI will be along imminently